rollup-node.md

```diff
@@ -230,23 +235,35 @@ The [previous section on L2 chain derivation][l2-chain-derivation] assumes linea
 also applicable for batch processing, meaning that any given point in time, the canonical L2 chain is given by
 processing the whole L1 chain since the [L2 chain inception].
 
-The L1 chain may occasionally re-organize, meaning the head of the L1 chain changes to a block that is not the child of
-the previous head but rather one of its "cousins" (i.e. the descendant of an ancestor of the previous head). In those
-case, the rollup driver must:
-
-1. Locate the *common ancestor*, a block that is an ancestor of both the previous and new head.
-2. Isolate the range of L1 blocks `]common ancestor, ..., new head]`.
-3. For each such block, call [`engine_forkchoiceUpdatedOPV1`] and [`engine_getPayloadV1`].
+> By itself, the previous section fully specifies the behaviour of the rollup driver. **The current section is
+> non-specificative** but shows how L1 re-orgs can be handled in practice.
+
+In practice, the L1 chain is processed incrementally. However, the L1 chain may occasionally re-organize, meaning the
+head of the L1 chain changes to a block that is not the child of the previous head but rather one of its "cousins" (i.e.
+the descendant of an ancestor of the previous head). In those case, the rollup driver must:
+
+1. Call [`engine_forkchoiceUpdatedOPV1`] for the new L2 chain head
+    - Pass `null` for the `payloadAttributes` parameter.
+    - Fill the [`ForkchoiceStateV1`] object according to [the section on the execution engine][calling-exec-engine], but
+      set `headBlockHash` to the hash of the new L2 chain head. `safeBlockHash` and `finalizedBlockHash` must be updated
+      accordingly.
+2. If the call returns `"SUCCESS"`, we are done: the execution engine retrieved all the new L2 blocks via [block sync].
+3. Otherwise the call returns `"SYNCING"`, and we must derive the new blocks ourselves. Start by locating the *common
+   ancestor*, a block that is an ancestor of both the previous and new head.
+4. Isolate the range of L1 blocks `]common ancestor, ..., new head]`.
```

Comment:
```suggestion
4. Isolate the range of L1 blocks `[common ancestor, ..., new head]`.
```
typo

Reply:
I wasn't sure if this was a typo, or meant to imply the common ancestor is excluded from the range.

Reply:
Not a typo, meant to indicate exclusion: https://en.wikipedia.org/wiki/Interval_(mathematics)#Notations_for_intervals
How'd you guys write that in high school? With parentheses? (e.g. `(a, b]`)

Will change to:

```suggestion
4. Isolate the range of L1 blocks from `common ancestor` (excluded) to `new head` (included).
```